dartsim/src/SDFFeatures.cc

```diff
@@ -273,6 +281,14 @@ static ShapeAndTransform ConstructSphere(
   return {std::make_shared<dart::dynamics::SphereShape>(_sphere.Radius())};
 }
 
+/////////////////////////////////////////////////
+static ShapeAndTransform ConstructCapsule(
+    const ::sdf::Capsule &_capsule)
+{
+  return {std::make_shared<dart::dynamics::CapsuleShape>(
+        _capsule.Radius(), _capsule.Length())};
```

Comment:
After playing with this a bit, I noticed that collisions between the capsule shape and a plane get unstable quickly. With 0.008 step size all other shapes collide reliably with a plane, but the capsule sinks :confused: The largest step size I could use without it sinking was 0.002.

Reply:
Should we ticket an issue for this and come back to it after the Edifice release?

Reply:
I suspect this is a DART internal issue and the only thing we could do would be to use a mesh instead, like we're doing for ellipsoids. I'm inclined not to ticket an issue until this starts affecting users.